### PR TITLE
Improve the error message

### PR DESCRIPTION
**Pull request by theonefx**
Signed-off-by: theonefx <chenxilzx1@gmail.com>

# Description

Improve the error message for when a state store has not been set for actor usage and actors are used

## Issue reference

Please reference the issue this PR will close: https://github.com/dapr/dapr/issues/4273

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by mukundansundar — CHANGES_REQUESTED** [review]
@theonefx check and change line 41. That is the err message which was posted in the bug that is fixed by this PR. #4273

**Review by artursouza — CHANGES_REQUESTED** [review]

File: pkg/messages/api_errors.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ const (
 	ErrMalformedRequestData = "can't serialize request data field: %s"
 
 	// State.
-	ErrStateStoresNotConfigured = "state store is not configured"
+	ErrStateStoresNotConfigured = "The state store is not configured to use the actor runtime. Have you set the - name: actorStateStore value: \"true\" in your state store component file?"
```
[the remark this answers is not in the document]
Comment (theonefx) [reply pkg/messages/api_errors.go]: done

**Review by daixiang0 — APPROVED** [review]

**Comment by yaron2** [comment]
@theonefx why did you close this?

**Comment by theonefx** [comment]
> @theonefx why did you close this?

Sorry, this is a misoperation ....

**Comment by theonefx** [comment]
@yaron2 
I resubmitted a PR there: https://github.com/dapr/dapr/pull/4533

**Pull request closed without merging**